src/test/CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
-if (PROJECT_IS_TOP_LEVEL)
```

Comment:
Notice that I just moved this one level up to have less different places where we check for inclusions.